libsolidity/formal/VariableUsage.cpp

```diff
@@ -33,7 +33,6 @@ VariableUsage::VariableUsage(ASTNode const& _node)
 			solAssert(declaration, "");
 			if (VariableDeclaration const* varDecl = dynamic_cast<VariableDeclaration const*>(declaration))
 				if (
-					varDecl->isLocalVariable() &&
 					identifier->annotation().lValueRequested &&
```

Comment:
This might not work correctly for tuple assignments, I created an issue for that: https://github.com/ethereum/solidity/issues/4141

Reply:
True, I wanted to check that after a similar comment from you